cmd/root.go

```diff
@@ -440,6 +420,8 @@ func NewCommand(opts ...Option) *Command {
 	localFlags.BoolP("help", "h", false, "Display help information for cloud-sql-proxy")
 	localFlags.BoolP("version", "v", false, "Print the cloud-sql-proxy version")
 
+	localFlags.StringVar(&c.conf.ConfigFilepath, "config-file", c.conf.ConfigFilepath,
```

Comment:
nit: &c.conf.Filepath seems feasible without the conf config duplication.

Reply:
agreed. conf.Config is a bit much. Changed it.